test-utils/state-viewer/src/cli.rs

```diff
@@ -104,13 +101,22 @@ impl StateViewerSubCommand {
 
 #[derive(Clap)]
 pub struct DumpStateCmd {
+    /// Optionally, can specify at which height to dump state.
     #[clap(long)]
     height: Option<BlockHeight>,
+    /// Dumps state records and genesis config into separate files.
+    /// Has reasonable RAM requirements.
+    /// Use for chains with large state, such as mainnet and testnet.
```

Comment:
I'd also mention:

if false - writes all the information into a single file, which is useful for smaller networks